protocol/lib/log/constants.go

```diff
@@ -0,0 +1,52 @@
+package log
+
+const (
+	SourceModuleKey = "source_module"
+	Error           = "error"
+)
+
+// Tag keys
+// Do not have anything generic in here. For example, `Status` is too vague
+// and can be clarified as `OrderStatus` or `DaemonHealthStatus`.
+const (
+	Module              = "module"
+	TxMode              = "tx_mode"
+	OperationsQueue     = "operations_queue"
+	Callback            = "callback"
+	BlockHeight         = "block_height"
+	Msg                 = "msg"
+	ProposerConsAddress = "proposer_cons_address"
+	Handler             = "handler"
+	Tx                  = "tx"
+	OrderHash           = "order_hash"
+	OrderStatus         = "order_status"
+	Subaccount          = "subaccount"
+	PerpetualId         = "perpetual_id"
+	MevMatches          = "mev_matches"
+	StackTrace          = "stack_trace"
+	Proposer            = "proposer"
+	PrunableBlockHeight = "prunable_block_height"
+
+	OrderSizeOptimisticallyFilledFromMatchingQuantums = "order_size_optimistically_filled_from_matching_quantums"
+	NewLocalValidatorOperationsQueue                  = "new_local_validator_operations_queue"
+	LocalValidatorOperationsQueue                     = "local_validator_operations_queue"
+)
+
+// Tag values
```

Comment:
why do these tag values use underscores (or backslashes)?

why are module names like `x/clob` defined in same const def as callbacks?

Reply:
To retain backwards compatibility with the existing tags. Existing tags are in underscore delimited except for the module names, which are `x/clob` and `x/vesting`.